imports/plugins/core/ui-navbar/client/components/navbar/navbar.js

```diff
@@ -8,11 +8,10 @@ Template.CoreNavigationBar.onCreated(function () {
 });
 
 Template.CoreNavigationBar.onRendered(function () {
-  const userId = Meteor.userId();
   currentRoute = Router.getRouteName();
   this.autorun(() => {
-    if (Accounts.findOne(userId)) {
-      if (!Accounts.findOne(userId).takenTour && Accounts.findOne(userId).emails[0]) {
+    if (Accounts.findOne(Meteor.userId())) {
```

Comment:
Is there a reason we have a lot of code repetition here?

Reply:
I had to go back to doing this because apparently the tour guide doesn't pop up for a newly register user until he/she refreshes the page.

Reply:
Ok. This works.